bundles/org.openhab.binding.wolfsmartset/README.md

```diff
@@ -0,0 +1,96 @@
+4. Enter account credentials and create the system thing
+   ![Enter account credentials](doc/images/step4.png "step 4 - Enter account credentials")
+5. Wait until the the systems show up in the inbox. Select system from inbox and create thing.
+   ![Select system from inbox](doc/images/inbox-system.png "step 5 - Select system from inbox")
+5. Wait until the the units show up in the inbox. Select unit from inbox and create thing
+   ![Select unit from inbox](doc/images/inbox-units.png "step 5 - Select unit from inbox")
+6. Add points or equipment to your model
+   ![Add points or equipment to your model](doc/images/channels-unit.png "step 6 - Add points or equipment to your model")
+## Supported Things
+
+- Account (bridge)
+- System (bridge)
+- Unit
+
+## Thing configuration
+
+### Account
+
+| Parameter       | Type    | Defaut | Description                                                         |
+|-----------------|---------|----------|---------------------------------------------------------------------|
+| username | text | | username to authenticate to www.wolf-smartset.de |
+| password | text  | | password to authenticate to www.wolf-smartset.de |
+| refreshIntervalConfiguration | integer | 10 | Specifies the refresh interval to refresh the configuration in minutes |
+| refreshIntervalValues | integer | 15 | Specifies time in seconds to refresh values |
+| discoveryEnabled | boolean | true | disable the Thing discovery |
+
+### System
+
+| Parameter       | Type    | Defaut | Description                                                         |
+|-----------------|---------|----------|---------------------------------------------------------------------|
+| systemId | integer | | System ID assigned to the system by WolfSmartset webiste |
+
+### Unit
+
+| Parameter       | Type    | Defaut | Description                                                         |
+|-----------------|---------|----------|---------------------------------------------------------------------|
+| unitId | integer | | The BundleId assigned to the unit by WolfSmartset webiste |
+
+## Tested WOLF-Devices
+
+| WOLF Equipment    | openhab Version | Used gateway  |
+|-------------------|-----------------|---------------|
+| CSZ (CGB and SM1) | 3.1             | WOLF Link Pro |
+| CGB-2             | 3.1             | WOLF Link home|
+
+Note: Please update this table if you did a successfull test
+
+## Supported Heating-Devices
+
+All devices able to be connected to www.wolf-smartset.de
+
+### Regarding documentation from WOLF
+https://www.wolf.eu/fileadmin/Wolf_Daten/Dokumente/FAQ/3065655_201711.pdf
+
+| Heating system                            | WOLF Link home        | WOLF Link pro      |
+|-------------------------------------------|-----------------------|--------------------|
+| Gas condensing boiler CGB-2, CGW-2, CGS-2 | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
I'm not sure whether emoticons work this way. In doubt i would use unicode emoticons.